attractions.html

```diff
@@ -0,0 +1,82 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Filmopolis Attractions</title>
+<!-- link css -->
+    <link rel="stylesheet" type="text/css" href="style.css"/>
```

Comment:
We should re-use @pmcclure852 global stylesheet to prevent duplicating codes
```suggestion
    <link rel="stylesheet" type="text/css" href="stylesheet/style.css"/>
```

Reply:
Ran into some issues when trying to put style.css into a stylesheet folder, the images wouldn’t load?

Reply:
Since the stylesheet is here  <link rel="stylesheet" type="text/css" href="stylesheet/style.css"/> I think you would have to change the path of the images. If you want to read more how, this [article](https://www.w3schools.com/html/html_filepaths.asp) is a good read.

For your images you'll change the paths to:
```css
.single-box:nth-child(1) .img-area{
    background-image: url(/images/ride1.jpg);
}
.single-box:nth-child(2) .img-area{
    background-image: url(/images/ride2.jpg);
}
.single-box:nth-child(3) .img-area{
    background-image: url(/images/ride3.jpg);
}
.single-box:nth-child(4) .img-area{
    background-image: url(/images/ride4.jpg);
}
```

Reply:
Thanks for sharing, I'll check it out!